RecipesLyricsMashup/RecipesLyricsMashupV2/RecipeLyricsMashup/RecipeLyricsMashup/Pages/Index.cshtml

```diff
@@ -3,9 +3,10 @@
 @{
     ViewData["Title"] = "Home page";
     var recipePuppyJson2 = ViewData["recipePuppyJson2"];
-    QuickTypeRecipe.Result[] recipePuppyJson = (QuickTypeRecipe.Result[]) ViewData["recipePuppyJson"];
+    QuickTypeRecipe.Result[] recipePuppyRecipies = (QuickTypeRecipe.Result[]) ViewData["recipePuppyRecipies"];
     //var geniusResults = ViewData["geniusResults"];
     QuickType.Hit[] geniusResults = (QuickType.Hit[])ViewData["geniusResults"];
+    var search = ViewData["SearchString"]; //Morgan added 

```

Comment:
Avoid putting discussion or conversation in source code comments.  We can tell you added this by looking at commit history, so stating it in code is redundant.